tests/src/test/scala-2.13+/cats/tests/ScalaVersionSpecific.scala

```diff
@@ -212,6 +216,34 @@ trait ScalaVersionSpecificAlgebraInvariantSuite {
     )
   }
 
+  // This version-specific instance is required since 2.12 and below do not have parseString on the Numeric class
+  implicit protected def eqFractional[A: Eq: Arbitrary]: Eq[Fractional[A]] = {
+    // This deprecated instance is required since there is not `ExhaustiveCheck` for any types for which a `Fractional`
+    // can easily be defined
+    import DeprecatedEqInstances.catsLawsEqForFn1
```

Comment:
Note that we need to use [this instance](https://github.com/typelevel/cats/blob/08d7accf2f5dddde86972da9aa9f89b99d777189/laws/src/main/scala/cats/laws/discipline/Eq.scala#L142) since there is no type for which both `ExhaustiveCheck` and `Fractional` are defined. I attempted to fix this in #3813 with a new `MiniFloat` type but that made the PR to big.